bqskit/ir/gates/reset.py

```diff
@@ -0,0 +1,27 @@
+"""This module implements the Reset class."""
+from __future__ import annotations
+
+from bqskit.ir.gates.constantgate import ConstantGate
+from bqskit.qis.unitary.unitary import RealVector
+from bqskit.qis.unitary.unitarymatrix import UnitaryMatrix
+
+
+class Reset(ConstantGate):
+    """Pseudogate to initialize the qudit to |0>."""
```

Comment:
"initialize" should be changed to "re-initialize" or "reset/initialize", so that it clarifies this can be performed in the middle of a circuit not just at the beginning.